scr/main/java/bittercode/model/User.java

```diff
@@ -0,0 +1,88 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class User implements Serializable {
+
+    private String emailId;
+    private String password;
+    private String firstName;
+    private String lastName;
+    private Long phone;
+    private Char address;
+    private int<UserRole> roles;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Incorrect data type for roles field</b></div>
<div id="fix">

The `roles` field is declared as `int&lt;UserRole&gt;`, which appears to be a syntax error. Consider using `List&lt;UserRole&gt;` instead, as it's used in the getter and setter methods.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -14,1 +14,1 @@
 -    private int<UserRole> roles;
 +    private List<UserRole> roles;
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/3#issuecomment-2434572244>#7fa0a4</a></i></small>
</div>


---
🚩 Problem result:
- [ ] This issue was flagged incorrectly